developer/src/kmc-kmn/test/kmw/constants.tests.ts

```diff
@@ -0,0 +1,152 @@
+/*
+ * Keyman is copyright (C) SIL Global. MIT License.
+ *
+ * Created by mcdurdin on 2025-10-16
+ *
+ * Key cap special values (such as "*Shift*") are defined in multiple modules;
+ * this data is not currently in a common module, as it would create unhelpful
+ * dependencies or require significant refactoring. So, instead, for now we just
+ * verify that the values line up. Note that the _imported_ files are copied in
+ * by build.sh before running the tests in order to avoid pathing issues.
+ *
+ * This is testing that the values in these files are equivalent:
+ *   developer/src/tike/xml/layoutbuilder/constants.js
+ *   web/src/engine/osk/src/specialCharacters.ts
+ *   developer/src/kmc-kmn/kmw-compiler/constants.ts
+ *   core/include/ldml/keyman_core_ldml.ts
+ *   developer/docs/help/reference/file-types/keyman-touch-layout.md
+ */
+import * as fs from 'node:fs';
+import * as path from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+import 'mocha';
+import {assert} from 'chai';
+
+import keymanWebSpecialCharacters from "./_imported_specialCharacters.js";
+import { CSpecialText17, CSpecialText14, CSpecialText10, CSpecialText17ZWNJ } from "../../src/kmw-compiler/constants.js";
+import { builder } from "./_imported_constants.js";
+import { constants as coreConstants } from "@keymanapp/ldml-keyboard-constants";

```

Comment:
I'm wondering if it would be better to name this `ldmlConstants` or `ldmlCoreConstants`? Even though they live in `core` we talk about them as `LDML`, so having `ldml` appear in the name might make it easier to understand.